src/prometheus.ts

```diff
@@ -22,12 +24,39 @@ export function registerGauge(name: string, help: string) {
   } catch (error) {}
 }
 
-export const requests = registerCounter("requests", "Total requests")!
+export const requests = registerCounter("requests", "Total requests")!;
 export const request_errors = registerCounter("request_errors", "Total failed requests")!;
 export const sink_requests = registerCounter("sink_requests", "Total sink requests")!;
-export const queue_size = registerGauge("queue_size", "Amount of promises being processed")!;
 
 export const entity_changes_inserted = registerCounter("entity_changes_inserted", "Total inserted entity changes")!;
 export const entity_changes_updated = registerCounter("entity_changes_updated", "Total updated entity changes")!;
 export const entity_changes_deleted = registerCounter("entity_changes_deleted", "Total deleted entity changes")!;
-export const entity_changes_unsupported = registerCounter("entity_changes_unsupported", "Total unsupported entity changes")!
\ No newline at end of file
+export const entity_changes_unsupported = registerCounter("entity_changes_unsupported", "Total unsupported entity changes")!;
+
+export function incrementInsertedEntityChanges(chain: string, moduleHash: string) {
+  entity_changes_inserted.inc();
+
+  const metricName = `entity_changes_inserted_${chain}_${moduleHash}`;
```

Comment:
🤔 ... don't think this is the right approach

We should be using Labels instead of defining the metric name with values

I also don't even think we need `incrementInsertedEntityChanges` or `incrementSinkRequests` methods, can use the regular `registerCounter`

Reply:
Updated this to use labels as suggested.